library/cmac.c

```diff
@@ -300,8 +299,10 @@ int mbedtls_cipher_cmac_update( mbedtls_cipher_context_t *ctx,
     /* If there is data left over that wasn't aligned to a block */
```

Comment:
The `mbedtls_cipher_update()` can't update it, because it does not have access to it.